packages/experimental/Shadow/src/shadowStyle.ts

```diff
@@ -1,16 +1,18 @@
 import { memoize } from '@fluentui-react-native/framework';
 import { ShadowToken } from '@fluentui-react-native/theme-types';
-import { ColorValue } from 'react-native';
+import { ColorValue, Platform } from 'react-native';
 
-// The blur property from the token is not the same as the shadow radius value,
+// For iOS/macOS, the blur property from the token is not the same as the shadow radius value,
 // it needs to be divided by 2 to achieve the same effect.
```

Comment:
Once https://github.com/microsoft/apple-ux-guide/pull/13 lands, we can link it here